setup.cfg

```diff
@@ -0,0 +1,30 @@
+[metadata]
+name = pyflow
+version = attr: pyflow/version.__version__
+author = European Centre for Medium-Range Weather Forecasts (ECMWF)
+author_email = software.support@ecmwf.int
+license = Apache 2.0
+license_files = LICENSE
+description = Create pythonic ecFlow suites
+long_description = file: README.md
+long_description_content_type=text/markdown
+url = https://pyflow-workflow-generator.readthedocs.io/en/
+
+[options]
+packages = find_namespace:
+include_package_data = True
+install_requires =
+    jinja2
+    requests
+
+[options.packages.find]
+include = pyflow*
+
+[options.extras_require]
+diagrams =
+    graphviz
+tests =
+    pytest
+all =
+    %(diagrams)s
+    %(tests)s
```

Comment:
For convenience, so that devs/contributors can just `pip install -e .[all]` and get all dependencies installed.